content/docs/ibm/install-kubeflow.md

```diff
@@ -0,0 +1,220 @@
++++
+title = "Install Kubeflow"
+description = "Instructions for deploying Kubeflow with the shell"
+weight = 4
++++
+
+This guide describes how to use the kfctl binary to
+deploy Kubeflow on IBM Cloud.
+
+## Prerequisites
+
+### Installing the IBM Cloud developer tools
+
+> If you already have `ibmcloud` installed with the `ibmcloud cs` plugin, you
+> can skip these steps.
+
+1.  Download and install the `ibmcloud` command line tool:
+    https://console.bluemix.net/docs/cli/index.html#overview
+
+1.  Install the `cs` (container-service) plugin:
+    ```bash
+    ibmcloud plugin install container-service -r Bluemix
+    ```
+1.  Authorize `ibmcloud`:
+    ```bash
+    ibmcloud login
+    ```
+
+### Setting environment variables
+
+To simplify the command lines for this walkthrough, you need to define a few
+environment variables.
+
+1.  Set `CLUSTER_NAME`, `CLUSTER_REGION`, and `CLUSTER_ZONE` variables:
+
+    ```bash
+    export CLUSTER_NAME=knative
+    export CLUSTER_REGION=us-south
+    export CLUSTER_ZONE=dal13
+    ```
+
+    - `CLUSTER_NAME` must be lowercase and unique among any other Kubernetes
+      clusters in this IBM Cloud region.
+    - `CLUSTER_REGION` can be any region where IKS is available. You can get a
+      list of all available regions via the
+      [IBM Cloud documentation](https://console.bluemix.net/docs/containers/cs_regions.html#regions-and-zones)
+      or via `ibmcloud cs regions`.
+    - `CLUSTER_ZONE` can be any zone that is available in the specified region
+      above. You can get a list of all avaible locations from the
+      [IBM Cloud documentation](https://console.bluemix.net/docs/containers/cs_regions.html#zones)
+      or by using `ibmcloud cs zones` after you set the region by using
+      `ibmcloud cs region-set $CLUSTER_REGION`.
+
+### Creating a IBM Cloud Kubernetes cluster
+
+To make sure the cluster is large enough to host all the Knative and Istio
+components, the recommended configuration for a cluster is:
+
+- Kubernetes version 1.15 or later
```

Comment:
The Kubernetes version here conflicts with the Kubeflow requirements:
https://www.kubeflow.org/docs/started/k8s/overview/#minimum-system-requirements